src/components/views/rooms/EventTile.tsx

```diff
@@ -906,6 +906,10 @@ export default class EventTile extends React.Component<IProps, IState> {
             permalink = this.props.permalinkCreator.forEvent(this.props.mxEvent.getId());
         }
 
+        const scrollToken = this.props.mxEvent.status
```

Comment:
```suggestion
        // we can't use local echoes as scroll tokens, because their event IDs change.
        // Local echos have a send "status".
        const scrollToken = this.props.mxEvent.status
```